Components/DayCalendar.js

```diff
@@ -9,50 +12,251 @@ var {
   ScrollView,
   View,
   Text,
+  PanResponder,
+  Image,
 } = React;
 
+type Event = {
+  title: string;
+  startAt: Date;
+  length: number;
+};
```

Comment:
No. It's not part of RN. [Flow](http://flowtype.org/) is a tool from facebook. Basically you can add type annotation to your js code, and let flow do type check for you. Here is an example from their website:
```
function foo(x) {
  return x * 10;
}
foo('Hello, world!');
```
Flow will raise an error: `string is not compatible with number`